### PR TITLE
Added flake8-simplify

### PR DESCRIPTION
**Pull request by colin99d**
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
           restore-keys: ${{ runner.os }}-linting-${{ hashFiles('**/poetry.lock') }}
 
       - run: |
-          pip install bandit black codespell mypy==0.982 safety pylint==2.15.2 packaging==22 ruff==0.0.243
+          pip install bandit black codespell mypy==0.982 safety pylint==2.15.2 packaging==22 ruff==0.0.246
```
Comment (jmaslek) [inline .github/workflows/linting.yml:48]: can you remove safety and packaging from this line.  They aren't needed and I keep forgetting

**Comment by piiq** [comment]
are you sure @colin99d ?

**Comment by colin99d** [comment]
Are you asking whether I am sure this is a good linter to have, or whether I am sure that the code works?

**Comment by piiq** [comment]
are you sure that you want 96 files where code related to the logic has been changed to be reviewed in 1 PR?=)

**Comment by piiq** [comment]
the linter is good, i'll take it=)

**Comment by colin99d** [comment]
95% of the changes are an autofixer. If you would prefer I do it another way, or break it off into autofixer and non autofixer PRs, let me know.

**Comment by colin99d** [comment]
https://github.com/charliermarsh/ruff#flake8-simplify-sim

**Review by jmaslek — APPROVED** [review]

**Pull request merged**